### PR TITLE
Add link to file naming Enhancer

### PR DESCRIPTION
**Pull request by juba0x00**
Hello @karlhorky ,

I hope you're doing well!

I apologize for the delay since the last closed pull request. I've taken your feedback into account and made the necessary adjustments.

I've created a separate repository specifically for the "Notion Backup Enhancer" script, focusing solely on enhancing the organization and readability of Notion backup files and directories, while building off the notion-backup project.

You can find the repository here: [Notion Backup Enhancer](https://github.com/juba0x00/notion-backup-enhancer).

Following your suggestion, I've updated the README.md of the notion-backup repository to include a link to the Notion Backup Enhancer repository.

Please let me know if there are any further concerns or feedback you'd like to address.

Thank you for your time and consideration.

Best regards,
juba0x00

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ The GitHub Actions workflow is scheduled to run once a day to:
 2. wait until each export is done
 3. download, unzip and commit the content from each export to the repository
 
+## Notion Backup Enhancer
```
Comment (karlhorky) [inline README.md:23]: this section should be more like "Similar Projects" or "Related" or "Other Notion Backup Projects" or "See Also" or something like this

check this example:

- https://github.com/paulirish/lite-youtube-embed#other-fast-youtube-embeds

the content of the section should be 1 single bullet point with a title, short description and URL of your project, no paragraphs or screenshots

File: README.md [hunk 2]
```diff
@@ -20,6 +20,22 @@ The GitHub Actions workflow is scheduled to run once a day to:
...
+- Human-Readable Naming: The script renames all files and directories within your Notion backup to human-friendly names, making it easier to understand and navigate your backup structure.
+- Link Editing: It updates links within your Notion content to reflect the renamed files and directories. This ensures that your links remain valid and functional after the renaming process.
+- Handling Repeated Names: In cases where there are duplicate names in your backup, the script appends a unique identifier to the filenames, preventing naming conflicts and maintaining data integrity.
+
+**[Notion Backup Enhancer](https://github.com/juba0x00/notion-backup-enhancer)**
+### Before Enhancer
+![Before Enhancer](https://github.com/juba0x00/notion-backup-enhanced-pr/assets/73611543/3e8989a4-db43-41f9-887e-0c298f9b7e8e)
+### After Enhancer
+![After Enhancer](https://github.com/juba0x00/notion-backup-enhanced-pr/assets/73611543/7758b27d-2806-4dd0-8e35-c4cf3c7a3ca0)
+
```
Comment (karlhorky) [inline README.md:38]: this section should be more like "Similar Projects" or "Related" or "Other Notion Backup Projects" or "See Also" or something like this

check these examples:

- https://github.com/paulirish/lite-youtube-embed#other-fast-youtube-embeds
- https://github.com/sindresorhus/p-map#related

the content of the section should be 1 single bullet point with a title, short description and URL of your project, no paragraphs or screenshots

Comment (juba0x00) [reply README.md]: Thanks @karlhorky for your recommendations.
I've updated it

**Review by karlhorky — APPROVED** [review]

**Comment by karlhorky** [comment]
Thanks!

**Pull request merged**